### ISSUE TITLE
 Compared with the indicators

### ISSUE THREAD

**Issue by wangmingaaaaa** [vios-s/vMFNet#2]
Hi, I would like to ask why your paper is not related to the Enhancing Pseudo Label Quality for Semi-Supervised Domain-Generalized in 2022 Compared with the indicators of Medical Image Segmentation, is it because the backbone is different?

**Comment by xxxliu95** [comment]
Hi, yes, the paper you mentioned used a pre-trained ResNet as the backbone.

See in the related work: 
Following \cite{liu2021semi}, Yao et al.\ \cite{yao2022enhancing} adopted a pre-trained ResNet \cite{he2016deep} as a backbone feature extractor and augmented the source data by mixing MRI images in the Fourier domain and employed pseudo-labelling to leverage the unlabelled data.

Different backbones affect the DG performance.
